taxcalc/tests/test_parameters.py

```diff
@@ -210,75 +211,40 @@ def test_json_file_contents(tests_path, fname):
                               param.get('indexed', False),
                               param.get('indexable', False))
             failures += fail + '\n'
-        # check that value_type is correct string
-        if not param['value_type'] in valid_value_types:
-            msg = 'param:<{}>; value_type={}'
-            fail = msg.format(pname, param['value_type'])
-            failures += fail + '\n'
-        # check that indexable param has value_type real
-        if param.get('indexable', False) and param['value_type'] != 'real':
-            msg = 'param:<{}>; value_type={}; indexable={}'
-            fail = msg.format(pname, param['value_type'],
+        # check that indexable param has value_type float
+        if param.get('indexable', False) and param['type'] != 'float':
+            msg = 'param:<{}>; type={}; indexable={}'
+            fail = msg.format(pname, param['type'],
                               param.get('indexable', False))
             failures += fail + '\n'
         # ensure that indexable is False when value_type is not real
-        if param.get('indexable', False) and param['value_type'] != 'real':
-            msg = 'param:<{}>; indexable={}; value_type={}'
+        if param.get('indexable', False) and param['type'] != 'float':
+            msg = 'param:<{}>; indexable={}; type={}'
             fail = msg.format(pname,
                               param.get('indexable', False),
                               param['value_type'])
             failures += fail + '\n'
-        # check that value_yrs is list
-        valueyrs = param['value_yrs']
-        assert isinstance(valueyrs, list)
-        # check all value_yrs values
-        cyr = first_year
-        for vyr in valueyrs:
-            assert vyr == cyr
-            cyr += 1
-        # check type and dimension of value
-        value = param['value']
-        assert isinstance(value, list)
-        assert len(value) == len(valueyrs)
-        # check that vi_name and vi_vals are consistent
-        viname = param.get('vi_name', '')
-        assert isinstance(viname, str)
-        vivals = param.get('vi_vals', [])
-        if viname == '':
-            assert vivals == []
-        else:
-            assert isinstance(vivals, list)
-            # check different possible vi_name values
-            if viname == 'MARS':
-                assert len(vivals) == 5
-            elif viname == 'EIC':
-                assert len(vivals) == 4
-            elif viname == 'idedtype':
-                assert len(vivals) == 7
-            elif viname == 'c00100':
-                pass
-            else:
-                assert viname == 'UNKNOWN vi_name VALUE'
-            # check length of each value row
-            for valuerow in value:
-                assert len(valuerow) == len(vivals)
```

Comment:
These checks are enforced by ParamTools on load and when converting the list of dicts to numpy arrays.